test/ng/directive/ngOptionsSpec.js

```diff
@@ -1138,13 +1138,13 @@ describe('ngOptions', function() {
       });
 
       scope.$apply(function() {
-        scope.selected = scope.arr[0];
+        scope.selected = {id: 10, label: 'ten'};
```

Comment:
We need to use a totally different object here (which is matched by the `id` property to the option) so that we can then change the value of the `id` property of the model without changing the `id` property on the option.